src/Tools/CodeGenerator/Plugins/OnnxRuntimePlugin.py

```diff
@@ -428,7 +457,7 @@ def CreateMacro(
                                 type_name=type_name,
                                 constraints=", ".join(
                                     [
-                                        '"{}"'.format(constraint)
+                                        '"tensor({})"'.format(constraint[:-2] if constraint[-2:] == "_t" else constraint)

```

Comment:
Change this to `... if constraint.endswith("_t") else ...`, which will prevent errors if we happen to see a constraint with only 1 char.

Reply:
Good catch. I fixed that everywhere else but missed that one.